radio/src/translations/cz.h

```diff
@@ -31,6 +31,11 @@
  *  \001 to \034     -extended spacing (value * FW/2)
  */
 
+#define TR_MIN_PLURAL2  2
+#define TR_MAX_PLURAL2  2
```

Comment:
Yes, that is why TR_MIN/MAX_PLURAL2 are per language.
Since I do not know majority of the languages :-), I left it to translators to change these values.